sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -56,20 +56,16 @@ export function useFormioDropdownLoader() {
     );
   };
 
-  // Retrieve the list of programs that have some
-  // offering to the locationId authorized for
+  // Retrieve the list of programs
   // a particular ionstitution.
-  const loadProgramsForLocationForInstitution = async (
+  const loadProgramsForInstitution = async (
```

Comment:
Same clarification here. Does it have any dependency?

Reply:
No this is part of the previous bug and I did fixed it